scipy/sparse/__init__.py

```diff
@@ -103,9 +103,16 @@
     7. dia_matrix: DIAgonal format
 
 To construct a matrix efficiently, use either dok_matrix or lil_matrix.
-The lil_matrix class supports basic slicing and fancy
-indexing with a similar syntax to NumPy arrays.  As illustrated below,
-the COO format may also be used to efficiently construct matrices.
+The lil_matrix class supports basic slicing and fancy indexing with a
+similar syntax to NumPy arrays. As illustrated below, the COO format
+may also be used to efficiently construct matrices. Despite their
+to NumPy arrays, it is **strongly discouraged** to use NumPy functions
```

Comment:
Ummm...that's really awkward.  I swore I fixed it (otherwise, I wouldn't have made that comment above).  It should be fixed for good now.